### DISCUSSION TITLE
How do I create a new record with a relation with an HTTP request?

### DISCUSSION THREAD

**Discussion by ghost** [pocketbase/pocketbase#1790] [Q&A] [answered]
How do I create a new record with a relation with an HTTP request?

`expand_testing` has a field relation which is related to `expand_testing_data`

What am I doing wrong?

```http

POST http://localhost:8090/api/collections/expand_testing/records
Authorization: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2NzY4NTI1MzQsImlkIjoiN2NjczIzbXM2ZzlmM21kIiwidHlwZSI6ImFkbWluIn0.10pkeDSI2CH_ZnBBQmhJk0vVhRWj6ZGV6P7nuPAmUSk
Content-Type: Application/Json

{
  "relation": ["pq4u5sgwubz7i7s"]
}
```

<img width="806" alt="Screenshot 2023-02-05 at 6 52 56 PM" src="https://user-images.githubusercontent.com/34663132/216857831-559f9d85-f417-4419-9f9d-d0a5c649c4c6.png">
<img width="820" alt="Screenshot 2023-02-05 at 6 53 33 PM" src="https://user-images.githubusercontent.com/34663132/216857981-b977ad17-3f5b-41e4-a864-191befeae1c4.png">

**Comment by ganigeorgiev** [comment, marked as answer]
What is the error you are getting?

Make sure that your request satisfies the `expand_testing` "Create API" rule (by default only admins are allowed).
